Golang requirement is actually >= 1.7, stated as 1.6 on Developer Guide

Go 1.6.0 or later is claimed to be Prequisite, however 1.6 goLang build attempts result in this error:
vendor/k8s.io/client-go/rest/request.go:21:2: cannot find package "context" in any of:

context package is added as of 1.7, hence effective minimum version of GO required to build helm is actually 1.7 and above.

--- a/docs/developers.md
+++ b/docs/developers.md
@@ -5,10 +5,9 @@ Helm and Tiller.
 
 ## Prerequisites
 
-- Go 1.6.0 or later
-- Glide 0.12.0 or later
-- kubectl 1.2 or later
-- A Kubernetes cluster (optional)
+- The latest version of Go
+- The latest version of Glide
+- A Kubernetes cluster w/ kubectl (optional)
 - The gRPC toolchain
 - Git
 - Mercurial

